reward/rewardManager.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package reward
+
+import "github.com/klaytn/klaytn/log"
+
+var logger = log.NewModuleLogger(log.Reward)
+
+type governanceHelper interface {
```

Comment:
I think it is not useful when an interface is not exposed. Is this intentional?
AFAIK, it is not exposed if the first character is a lower case.

Reply:
This interface is for avoiding cycle to use governance object in reward package.
I made it internal as the interface is only used in reward package.

Reply:
@kjhman21 would it be okay to merge? or Do you think it needs to be external?

Reply:
No, in this case, please go forward. Thanks for the information.